src/services/queries/Users.js

```diff
@@ -14,18 +15,25 @@ export const getAllUsersQuery = gql`
   }
 `;
 
-export const getUserSessionQuery = gql`
-  query GetUserSession($password: String = "", $user_code: String = "") {
+export const USER_LOGIN_QUERY = gql`
+  ${userInfo}
+  query GetUserSession($password: String!, $member_code: String!) {
     users(
-      where: { password: { _eq: $password }, username: { _eq: $user_code } }
+      where: {
+        password: { _eq: $password }
+        member_code: { _eq: $member_code }
+      }
     ) {
-      username
-      type
-      password
-      is_active
-      id
-      created_at
-      avatar_url
+      ...userInfo
+    }
+  }
+`;
+
+export const USER_BY_ID_QUERY = gql`
+  ${userInfo}
+  query getUsers($id: uuid!) {
+    users(where: { is_active: { _eq: true }, id: { _eq: $id } }) {
```

Comment:
You can try to use user_by_pk to avoid bringing an array

Reply:
Of course, thanks !